docs/development.rst

```diff
@@ -30,7 +30,7 @@ Install Locally
 
     pip install -r requirements-dev.txt
     npm install
-    npm link stylus yuglify
+    npm link stylus autoprefixer-stylus yuglify
```

Comment:
We don't really need to link the package commands. They're only used during the build step, and if we need to run them locally, we can run `./node_modules/.bin/<command>`. That feels like a cleaner isolation than linking this project's commands into the global symlink?